js/templates/modals/purchase/actionBtn.html

```diff
@@ -45,3 +45,8 @@ <h3>
     <span class="js-closeNote"><%= ob.polyT('purchase.closeNote') %></span>
   <% } %>
 </div>
+<div class="padSm txSm txCtr">
```

Comment:
Did you forget to add `purchase.scamWarning` to the translation file? I'm not seeing it.

Reply:
done